lib/autoload/active_record/associations/preloader/manual_has_many_through.rb

```diff
@@ -0,0 +1,5 @@
+class ActiveRecord::Associations::Preloader
+  class ManualHasManyThrough < HasMany
```

Comment:
It looks that `HasMany` works well to `HasManyThrough`, too. I'm also trying to 'hack' the `HasManyThrough` preloader, but I think it is harder than `HasMany`. Because there is no such function like `records_for` which can be used to inject our records. Do you have any idea about this problem? @lowjoel